src/include/job.h

```diff
@@ -847,6 +846,19 @@ task_find	(job		*pjob,
 #define JOB_STATE_FINISHED	9
 
 
+#define JOB_STATE_LTR_BEGUN			'B'
+#define JOB_STATE_LTR_EXITING		'E'
+#define JOB_STATE_LTR_FINISHED		'F'
+#define JOB_STATE_LTR_HELD			'H'
+#define JOB_STATE_LTR_MOVED			'M'
+#define JOB_STATE_LTR_QUEUED		'Q'
+#define JOB_STATE_LTR_RUNNING		'R'
+#define JOB_STATE_LTR_SUSPENDED		'S'
```

Comment:
Looks like this (and 'U') are new state? is this external change? if yes then don't we need design for this?

Reply:
No, they are existing states